azure-devops/javascript/abstracts/corepack-node18.19.0.yaml

```diff
@@ -12,6 +16,6 @@ steps:
   - task: 'Cache@2'
     inputs:
       key: "$(Agent.JobName)|yarn.lock"
-      path: 'node_modules'
+      path: '$(Agent.TempDirectory)/OpenSearch-Dashboards/plugins/${{ parameters.PLUGIN_NAME }}/node_modules'
```

Comment:
I think this path should be passed via parameter, because this is a generic template to be used in other projects. Not just in this OSD one.

Reply:
THIS PATH! Should be passed via Parameters. Please, because this is a generic template file. The WHOLE PATH, not just the app name.